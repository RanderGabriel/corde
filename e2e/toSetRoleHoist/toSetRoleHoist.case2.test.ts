import Utils from "../testUtils";
import { runCLI } from "../cliRunner";

it("should change role color", async () => {
  const command = Utils.buildCommandWithConfigPath("toSetRoleHoist", "bot_case2.test.ts");
  const results = await cli.exec(command);
  expect(results.statusCode).toEqual(1);
  expect(results.stdout).toMatchSnapshot();
});
