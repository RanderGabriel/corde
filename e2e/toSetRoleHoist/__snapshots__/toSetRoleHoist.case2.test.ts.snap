// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should change role color 1`] = `
"
 FAIL   e2e\\\\toSetRoleHoist\\\\__cordeTest__\\\\bot_case2.test.ts   100ms
   x <empty test name>   100ms
    
    expected: hoist to be true
    received: role hoist was not updated

    at toSetRoleHoist e2e\\\\toSetRoleHoist\\\\__cordeTest__\\\\bot_case2.test.ts:8:46
    at lib\\\\src\\\\clausures\\\\test.js:31:13
    at Queue.executeAsync (lib\\\\src\\\\data-structures\\\\queue.js:99:27)

Test Files: 1 failed. 1 total
Tests:      1 failed. 1 total
Time:       100ms
"
`;
